CMakeLists.txt

```diff
@@ -62,17 +57,17 @@ option(DEBUG_MMS_CLIENT "Enable MMS CLIENT printf debugging" OFF)
 #				 DEBUG_IED_CLIENT DEBUG_MMS_SERVER DEBUG_MMS_CLIENT)
 
 include_directories(
-    ${CMAKE_CURRENT_BINARY_DIR}/config
-    ${CMAKE_CURRENT_LIST_DIR}/src/common/inc
-    ${CMAKE_CURRENT_LIST_DIR}/src/goose
-    ${CMAKE_CURRENT_LIST_DIR}/src/sampled_values
-    ${CMAKE_CURRENT_LIST_DIR}/src/hal/inc
-    ${CMAKE_CURRENT_LIST_DIR}/src/iec61850/inc
-    ${CMAKE_CURRENT_LIST_DIR}/src/iec61850/inc_private
-    ${CMAKE_CURRENT_LIST_DIR}/src/mms/inc
-    ${CMAKE_CURRENT_LIST_DIR}/src/mms/inc_private
-    ${CMAKE_CURRENT_LIST_DIR}/src/mms/iso_mms/asn1c
-    ${CMAKE_CURRENT_LIST_DIR}/src/logging
+    config
```

Comment:
What is the problem with ${CMAKE_CURRENT_LIST_DIR}. Shouldn't be kept?

Reply:
I think we can remove `${CMAKE_CURRENT_LIST_DIR}` here because a relative path is sufficient.

I was wondering why you used `${CMAKE_CURRENT_LIST_DIR}` instead of `${CMAKE_CURRENT_SOURCE_DIR}`.
Further down in the same file you set the include directories without the absolute path:

```cmake
IF(MSVC)
include_directories(
    src/vs
)
ENDIF(MSVC)
```



Heres the corresponding part of the documentation:

```
include_directories([AFTER|BEFORE] [SYSTEM] dir1 [dir2 ...])
```
> Add the given directories to those the compiler uses to search for include files. Relative paths are interpreted as relative to the current source directory.